source/math/analyse_patterns.json

```
@@ -261,5 +261,22 @@
       "critical",
       "stationary"
     ]
+  },
+  "monotonicity": {
+    "patterns": {
+      "^monotonic(ity)?[ ]+(of[ ]+)?((a|the)[ ]+)?(function[ ]+)?(.+)$": [
+        "6"
+      ],
+      "^type[ ]+(of[ ]+)?((a|the)[ ]+)?(function[ ]+)?(.+)$": [
+        "5"
+      ]
+    },
+    "keywords": [
+      "the",
+      "function",
+      "monotonicity",
+      "monotonic",
+      "type"
+    ]
   }
 }
```

```suggestion
}

```
No newline, maybe it needs

---

No, really, Github is a little dumb